examples/trt-external-payload-cluster-density.yaml

```diff
@@ -35,6 +35,28 @@ tests :
         - "[Jira: kube-apiserver]"
       direction: 1
 
+    - name: multusCPU
+      metricName : containerCPU
+      labels.namespace.keyword: openshift-multus
+      metric_of_interest: value
+      agg:
+        value: cpu
+        agg_type: avg
```

Comment:
This averages all pods and containers in the namespace .. do we want to focus on a container specifically? I think kube-rbac-proxy would be weighing it down